test/v2/integration/trading.ts

```diff
@@ -0,0 +1,45 @@
+import { coroutine as co } from 'bluebird';
+import should = require('should');
```

Comment:
`import * as should from 'should';`

`import .. = require()` should only be used for compatibility with default or callable exports.